### PR TITLE
RestyResolver Example

### PR DESCRIPTION
**Pull request by kleijnweb**

### CODE REVIEW

File: examples/restyresolver/api/pets.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+pets = {}
+
+
+def post(pet):
+    count = len(pets)
+    pet['id'] = count + 1
+    pet['registered'] = rfc3339(time.time())
+    pets[pet['id']] = pet
+    return Response(json.dumps(pet), status=201, mimetype='application/json')
```
Comment (hjacobs) [inline examples/restyresolver/api/pets.py]: any reason you are manually doing JSON serialization?

I would expect to just use `return pet`

Comment (kleijnweb) [inline examples/restyresolver/api/pets.py]: Ah, I can do "return pet, 201", no?

Comment (hjacobs) [inline examples/restyresolver/api/pets.py]: yes, exactly, see https://github.com/zalando/connexion/blob/master/connexion/decorators/decorator.py#L41

**Comment by hjacobs** [comment]
:+1: thanks!

**Pull request merged**